src/lib/syncable_ledger/test_mask.ml

```diff
@@ -0,0 +1,108 @@
+open Core
+
+(* Input for testing the interaction between merkle *)
+
+module Make (Input : sig
+  val depth : int
+
+  val mask_layers : int
+end) =
+struct
+  open Merkle_ledger_tests.Test_stubs
+
+  module Hash = struct
+    type t = Hash.t [@@deriving sexp, hash, compare, bin_io, eq]
+
+    type account = Account.t
+
+    let merge = Hash.merge
+
+    let hash_account = Hash.hash_account
+
+    let to_hash = Fn.id
+
+    let empty_account = hash_account Account.empty
+  end
+
+  module Maskable_and_mask =
+    Merkle_ledger_tests.Test_mask.Make_maskable_and_mask_with_depth (Input)
+
+  module L = struct
```

Comment:
`L`  could be `Ledger` or even `Layered_masks_ledger` to be more informative

Reply:
I can change that for all the test inputs. Yah, the module type for the test input are all abbreviations